capstone/src/components/User/Experience/Experience.jsx

```diff
@@ -42,13 +41,16 @@ export default function Experience({ setIsLoggedIn, isLoggedIn }) {
     } else if (username != null) {
       Promise.all([getExperienceInfo(username), getAllFeedbackInfo()]).then(
         ([expRes, feedbackRes]) => {
+          setExperienceData(expRes.data);
           // If there are no values for a field
           // send to that page to promote submission.
           // Otherwise, just set values in state
-          if (Object.keys(expRes.data).every((experienceField) => !!experienceField)) {
+          if (
+            !(Object.keys(expRes.data).every(
```

Comment:
you can also use `Object.entries` if you need both key and value.

Reply:
Thanks for that comment! Just implemented that